### PR TITLE
Platform Business APP- CCT CHAT BOT Test cases Automated ~Chaudhryma

### PR DESCRIPTION
**Pull request by Mariachaudhry**

### CODE REVIEW

**Review by Mariachaudhry — COMMENTED** [review]
Reviewed

**Review by sarwarahmed1 — APPROVED** [review]

File: src/test/java/ServiceNow/PlatformBusinessApps/CCT_CHAT_BOT/Playwright/Features/CCT_CHAT_BOT.feature [hunk 1]
```diff
@@ -21,4 +21,28 @@ Feature: CCT CHAT BOT Scenarios
...
+    And User selects "Post-Doctoral" for Q2 regarding career stage
+    And selects "USA" for Q3 about research training location
+    And selects "Yes" for Q4 "Are you a citizen of the USA, a noncitizen national, or a permanent resident?"
+    When the user selects "Yes" for Q5 "Do you consider yourself to belong to an underrepresented group in the biomedical research workforce?"
+    Then the "NCI Mentored Research Scientist Development Award (K01)" link should be displayed at the top of the funding list
+    And the link to "NCI Mentored Research Scientist Development Award (K01)" as "https://www.cancer.gov/grants-training/training/funding/k01" should be clickable
+    And the "NCI Mentored Research Scientist Development Award (K01)" link should open in a new tab
+
+  @SNOWPROJ-132 @Chaudhryma @Chaudhryma @Regression @playwright
+  Scenario: Test 2:Add K01 to the CCT Find Funding Chatbot
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/CCT_CHAT_BOT/Playwright/Features/CCT_CHAT_BOT.feature:39]: You could add a space between colon (:) and Add on line 39

**Review by hamidnesar — APPROVED** [review]
Approved!

**Review by georgeb79 — APPROVED** [review]

**Review by katm007 — APPROVED** [review]
Approved!

**Review by matakevin — APPROVED** [review]
LGTM

**Pull request merged**